MIP0/mip0.md

```diff
@@ -301,153 +263,82 @@ Entries into this list should follow the following template:
-- Must have experience engaging with different stakeholders in the community in all the different venues the community uses for communications including chat rooms, forums and video conference calls.
-- Should be confident in expressing themselves in each of the different venues the community uses for communications including chat rooms, forums and video conference calls.
-- Should have an interest in governance mechanisms used presently and historically across the world.
-
-**Removal**
-
-A Governance Facilitator should be considered for removal if they are:
--   Not adequately performing their defined duties
--   Absent from their duties for a prolonged period
--   Displaying biased or malicious behaviour
--   Expressing unwillingness to continue in their role.
-    
-The removal process begins once the community has agreed on the reasoning for removal. This process must be communicated publicly via the forums in order to provide complete transparency. **The Governance Facilitator will then be removed from the following channels:**
--   Github
--   RocketChat
--   Forums
+- Should encourage the community to come to consensus over the least objectionable option(s) rather than treating decision-making as a competition where a subset of the community must end up disappointed in the outcome.
+- Should work to bring the governance community together on divisive topics and to prevent political polarisation and demagoguery.
 
 ---
 
-### MIP0c12: Core Personnel Onboarding
+### MIP0c12: MIP Set Management
 
-MIP0c12 is a Process MIP component that allows the onboarding of core personnel using a subproposal. MIP0c12 subproposals have the following parameters:
--   **Feedback Period**: 3 months
--   **Frozen Period**: 1 month
+Some unitary structures and behaviors must be analyzed into separate parts across more than one MIP. When these MIPs are tightly interwoven, they constitute a MIP Set.
 
-MIP0c12 subproposals must use the template located at  **[MIP0c12-Subproposal-Template.md](MIP0c12-Subproposal-Template.md)**. This template is considered ratified once this MIP moves to Accepted status.
+#### Tagging
 
----
+For any MIP Set, its constituent MIPs must all share a common tag. The tag should be defined by MIP Authors and/or MIP Editors and should comply with the following format:
 
-### MIP0c13: Core Personnel Offboarding
+`descriptive short name-mipset`
 
-MIP0c13 is a Process MIP component that allows the removal of core personnel using a subproposal. MIP0c13 subproposals have the following parameters:
+This makes it easier to keep track of which MIPs act as a modular whole.
 
--   **Feedback Period**: 0 days
--   **Frozen Period**: 0 days
+MIP Sets are usually introduced as such by MIP Authors, but if MIP Editors find that a group of formally loose MIPs in the RFC phase satisfy the conditions, they can tag these MIPs together into a formal MIP Set. When in doubt, the ultimate criteria should be usefulness and distinctness of unity.
```

Comment:
'formally loose'? Sounds strange. Did you mean formerly (as in 'was loose') or 'formally loose' (as in 'loose in a structured way')

Reply:
'usefulness and distinctiveness of unity' is also a bit of a mouthful. Maybe try to simplify?

Reply:
Yep, it's a bit confusing. I kind of meant de jure loose (in spite of de facto bound)